src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -223,6 +229,31 @@ namespace AppInstaller::Manifest
                 {
                     resultErrors.emplace_back(ManifestError::FieldNotSupported, "ProductId");
                 }
+
+                // Ensure that each URL has a one to one mapping with a Sha256 and
+                // warn if a Sha256 has a one to many mapping with a URL
+                if (!installer.Url.empty() && !installer.Sha256.empty())

```

Comment:
Should we do this only when fullValidation is true? The validation pipeline uses fullValidation. Some other manifest read scenarios do not use fullValidation to save time.

Reply:
I think that may make sense